service/storage_service.cc

```diff
@@ -20,6 +20,10 @@
 #include "db/system_distributed_keyspace.hh"
 #include "db/consistency_level.hh"
```

Comment:
> With the default commitlog segment size, `mutation_size_threshold` will be 4 MB.
In large clusters e.g. 100 nodes, 64 shards per node, 256 vnodes cdc generation
data can reach the size of 30 MB, thus

> In a multi-DC cluster with 100ms latencies between DCs, which means ~200ms for
each command, because we first perform a barrier in start_operation and then commit
the command. Thus this operation will take about 1.6s which is incomparably shorter
than bootstrapping operation.

The "thus" in the first paragraph ends it abruptly, is something missing there?

BTW. bootstrapping is quick if there is no data in the cluster - but usually if one has 100 nodes they have tons of data, so indeed streaming/repair will take much longer (hours/days) and additional 1.6s is not a problem.

Reply:
> The "thus" in the first paragraph ends it abruptly, is something missing there?

Yes, thanks, I missed it. Now it is fixed.